src/client/components/matchResults/scoreKeepByInnings/batterStats.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import Divider from '@material-ui/core/Divider';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelActions from '@material-ui/core/ExpansionPanelActions';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import IconButton from '@material-ui/core/IconButton';
+import Typography from '@material-ui/core/Typography';
+import AddCircleOutline from '@material-ui/icons/AddCircleOutline';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import RemoveCircleOutline from '@material-ui/icons/RemoveCircleOutline';
+
+import { buttonText, battersConfig } from 'client/constants/base';
+
+const renderBatter = batterNumbers => {
+  const batters = [];
+  for (let i = 0; i < batterNumbers; i += 1) {
+    batters.push(
+      <ExpansionPanel key={`batter-state-#${i}`}>
+        <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+          <div>
+            <Typography>
+              {`No.${(i % battersConfig.battersInARow) + 1}`}
+            </Typography>
+          </div>
+        </ExpansionPanelSummary>
+        <ExpansionPanelDetails>
+          <div>
+            <Typography>Result</Typography>
+          </div>
+        </ExpansionPanelDetails>
+        <Divider />
+        <ExpansionPanelActions>
+          <Button size="small">{buttonText.batterStat.cancel}</Button>
+          <Button size="small" color="primary">
+            {buttonText.batterStat.save}
+          </Button>
+        </ExpansionPanelActions>
+      </ExpansionPanel>
+    );
+  }
+
+  return batters;
+};
+
+const BatterStats = ({ defaultBatters }) => {
+  const [batters, setBatters] = useState(defaultBatters);
+  const handleAddClick = () => setBatters(batters + 1);
```

Comment:
I keep the number to `9` because I aim to follow the batting order, so the final goal is that if in first inning, we only have 4 batters hitting, then next innings it'll start from batter `no.5`.
BTW, it just my fundamental idea now, it can always have alternative solution, feel free to give me any ideas! Thanks :)

Reply:
Oh! I see! Understand the behavior here; then it looks fine to me!

Reply:
In the future, I'll let user to input their batters' name so that it might be more clear to understand